radioactive-decay/content.md

```diff
@@ -1,44 +0,0 @@
-# Radioactive Decay
```

Comment:
If this PR is approved and merged, it will add and then delete the radioactive decay lesson. It needs to be cleaned up (the easiest way is to restart it in a new branch that branches off of master).